src/table.ts

```diff
@@ -1886,74 +1984,39 @@ class Table extends common.ServiceObject {
     }
 
     delete json.createInsertId;
+    delete json.maxAttempts;
     delete json.raw;
+    delete json.schema;
 
-    let schema: string | {};
-
-    if (options.schema) {
-      schema = options.schema;
-      delete json.schema;
-    }
-
-    const createTableAndRetry = () => {
-      this.create(
-        {
-          schema,
-        },
-        (err, table, resp) => {
-          if (err && err.code !== 409) {
-            callback!(err, resp);
-            return;
-          }
-
-          setTimeout(() => {
-            this.insert(rows, options, callback!);
-          }, 60000);
-        }
-      );
-    };
-
-    this.request(
-      {
-        method: 'POST',
-        uri: '/insertAll',
-        json,
-      },
-      (err, resp) => {
-        if (err) {
-          if ((err as common.ApiError).code === 404 && schema) {
-            setTimeout(createTableAndRetry, Math.random() * 60000);
-          } else {
-            callback!(err, resp);
-          }
-          return;
-        }
+    const [resp] = await this.request({
```

Comment:
@steffnay it appears that this return from `request` was refactored by @callmehiphop, where request is now returning an array (tuple) containing the response, but previously this was not the case (as you can see by this diff). It appears this was a good fix per `@google-cloud/common`'s `service-object.d.ts`:
```
export declare type RequestResponse = [Metadata, r.Response];
```
But I'm having trouble confirming if this is really the case. Can you verify?

Reply:
@zamnuts Yeah, it looks like that's correct.